execution/executor-types/src/in_memory_state_calculator.rs

```diff
@@ -245,7 +256,10 @@ impl InMemoryStateCalculator {
                     &mut self.usage,
                     (*write_set).clone(),
                 )?;
-                self.updates_after_latest.extend(state_updates.into_iter());
+                state_updates.iter().for_each(|(k, v)| {
```

Comment:
nit: consolidate here with line 278 - 280 using a util function? would be easier to read as well.

Reply:
Done